code/modules/virus2/isolator.dm

```diff
@@ -74,8 +74,11 @@
 		
 						var/datum/weakref/A = B.data["donor"]
 						var/mob/living/carbon/human/D = A.resolve()
+						var/Z
+						if(D)
+							Z = 1
 						pathogen_pool.Add(list(list(\
-							"name" = "[D.get_species()] [B.name]", \
+							"name" = "[Z ? D.get_species() : "unknown"] [B.name]", \

```

Comment:
...why not just directly check for D here?

Reply:
What Lohikar said. `"name" = "[D ? D.get_species() : "unknown"] [B.name]",`